net-misc/dleyna-server/dleyna-server-0.5.0.ebuild

```diff
@@ -29,6 +29,9 @@ DEPEND="${RDEPEND}
 	virtual/pkgconfig
 "
 
+# Bug 597952
+PATCHES=( "${FILESDIR}"/${P}-fix-references-to-GUPnPContextManager.patch )
+
 src_install() {
 	default
 	prune_libtool_files
```

Comment:
Package does not install static libraries, nor does it have a USE flag that would make it so. I replaced the {{prune_libtool_files}} and {{inherit eclass}}